### DISCUSSION TITLE
how to get expand data in javascript

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3127] [Q&A] [answered]
can you share the syntax for getting expand field values I am using below syntax
`console.log(data.id, data.getString("status"), data.get("track_id"));`
here track_id field has relationship with other collections. i want to use expand on track_id

**Comment by ganigeorgiev** [comment, marked as answer]
If you haven't already, you can check https://pocketbase.io/docs/js-records/#programmatically-expanding-relations

If that doesn't help, please provide a more complete code sample showing what `data` is and how it was loaded.
